var-core/src/main/java/io/varhttp/ContentTypes.java

```diff
@@ -152,4 +153,9 @@ public int hashCode() {
 			return result;
 		}
 	}
+
+	@Override
+	public String toString() {
```

Comment:
When is it used?

Reply:
In places java is using it. Any concatenation with a string will trigger it, e.g. exception text